Samples/Samples/View/CalendarEventPage.xaml.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using Xamarin.Essentials;
+using Xamarin.Forms;
+
+namespace Samples.View
+{
+    public partial class CalendarEventPage : BasePage
```

Comment:
Check the code behind for other pages and see whether they specify the base class (ie BasePage). My preference is to remove this from the codebehind but we need to be consistent with what's on other pages

Reply:
All other pages seem to inherit from BasePage